package/src/components/ChannelPreview/hooks/useChannelPreviewDisplayPresence.ts

```diff
@@ -44,11 +44,10 @@ export const useChannelPreviewDisplayPresence = <
   );
   const channelMemberOnline = members.some((member) => member.user?.online);
 
-  const [displayPresence, setDisplayPresence] = useState(false);
-
-  useEffect(() => {
-    setDisplayPresence(getChannelPreviewDisplayPresence(channel, client));
-  }, [channel, channelMemberOnline, client]);
+  const displayPresence = useMemo(() => {
+    return getChannelPreviewDisplayPresence(channel, client);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
We shouldn't need this, why is `eslint` complaining ? If we do, let's try to fix the hook instead of disabling the rule

Reply:
It is for the additional `channelMemberOnline` dep

Reply:
That should not be the case, that one was there before too and it was also the dependency of a hook.

Reply:
Yeah, its weird that it didn't complain before

Reply:
Regardless, the issue with the dependencies should be fixed; we should not introduce any more of the `eslint-disable` comments unless we have a very very good reason to do so

Reply:
This has been fixed now.